src/pyk/proof/reachability.py

```diff
@@ -296,21 +345,54 @@ def summary(self) -> Iterable[str]:
 
 class APRProver:
     proof: APRProof
+    kcfg_explore: KCFGExplore
     _is_terminal: Callable[[CTerm], bool] | None
     _extract_branches: Callable[[CTerm], Iterable[KInner]] | None
     _abstract_node: Callable[[CTerm], CTerm] | None
 
+    main_module_name: str
+    some_dependencies_module_name: str
+    all_dependencies_module_name: str
```

Comment:
```suggestion
    main_module_name: str
    dependencies_module_name: str
    circularities_module_name: str
```

I think this naming is more clear, but up to your.